enf/enf_test.go

```diff
@@ -1,11 +1,23 @@
 package enf
 
 import (
+	"fmt"
 	"net/http"
 	"net/http/httptest"
+	"reflect"
+	"strings"
 	"testing"
 )
 
+type TestingParameters struct {
+	Path             string
+	Body             interface{}
+	ExpectedResponse string
```

Comment:
The name `ExpectedResponse` is confusing.

Normally tests this sort fo flow:
```
result = method_under_test(...0);
expected = "the expected result";
test_equal(result, expected);
```

The most developers will expect (no pun intended) that `ExpectedResponse` would be the ground-truth value being compared to the actual result.   In fact, `Want` is the ground-truth value and `ExpectedResponse` is the hard-coded response for the mock API server.

So let's
1) rename `ExpectedResponse` to something like `ResponseBodyMock`
1) rename `Want` to `Expected`

Those names are a bit longer, but should make the tests themselves easier for other developers to read.